project/LibDependencies.scala

```diff
@@ -1,47 +1,61 @@
-import sbt._
+import sbt.{ModuleID, _}
 import PlayCrossCompilation._
-import sbt.ModuleID
 import play.sbt.PlayImport.ws
 
 object LibDependencies {
   lazy val govukFrontendVersion = "3.11.0"
 
-  lazy val libDependencies: Seq[ModuleID] = dependencies(
-    shared = {
+  private val Play28Version = "2.8.7"
```

Comment:
Since these versions are setup in plugins.sbt, can you not just use `PlayVersion.current`? Ifo so you could move the following into shared:
```
"com.typesafe.play" %% "play"            % PlayVersion.current,
"com.typesafe.play" %% "filters-helpers" % PlayVersion.current
```

Reply:
done.